src/vault.ts

```diff
@@ -42,9 +42,14 @@ export abstract class BaseVault<StorableItem> {
      * @param item {StorableItem}
      */
     public store(item: StorableItem): void {
+        let stringifiedItem: string;
         this.contents = item;
 
-        const stringifiedItem = !isEmpty(item) ? JSON.stringify(item) : '';
+        if (isNumber(item)) {
+            stringifiedItem = JSON.stringify(item);
+        } else {
+            stringifiedItem = !isEmpty(item) ? JSON.stringify(item) : '';
+        }
```

Comment:
I _think_ this might be cleaner and more readable, but I'm not adamant about it.

```suggestion
        if (isNumber(item) || !isEmpty(item)) {
            stringifiedItem = JSON.stringify(item);
        } else {
            stringifiedItem = '';
        }
```